qmk_porting/keyboards/m8rev1/m8rev1.c

```diff
@@ -0,0 +1,209 @@
+// Copyright 2023 alin elena <https://github.com/alinelena> <https://gitlab.com/drFaustroll>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+#include "m8rev1.h"
```

Comment:
`QMK_KEYBOARD_H` is set to `m8rev1` by the building system, see[https://github.com/O-H-M2/qmk_port_ch582/blob/via/qmk_porting/qmk_porting.cmake#L72](url)